<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportWizard</class>
 <widget class="QWizard" name="ImportWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/tulip/gui/icons/logo32x32.png</normaloff>:/tulip/gui/icons/logo32x32.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="options">
   <set>QWizard::ExtendedWatermarkPixmap|QWizard::HaveFinishButtonOnEarlyPages|QWizard::HaveHelpButton|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <property name="titleFormat">
   <enum>Qt::RichText</enum>
  </property>
  <property name="subTitleFormat">
   <enum>Qt::RichText</enum>
  </property>
  <widget class="QWizardPage" name="choosePluginPage">
   <property name="styleSheet">
    <string notr="true">QListView, QListWidget {
border: 0px;
}

QListWidget::item {
padding: 6px;
}

#categoryFrame, #parametersFrame {
border: 1px solid #C9C9C9;
}

#categoryLabel, #parametersLabel {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
stop:1 rgba(0,0,0,0),
stop:0 #F0F0F0);
border: 0px;
}</string>
   </property>
   <property name="title">
    <string>&lt;img src=&quot;:/tulip/gui/icons/64/document-import.png&quot; width=32 height=32/&gt; Graph import Wizard</string>
   </property>
   <attribute name="pageId">
    <string notr="true">0</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Import a graph hierarchy into your project. First, select an import method, then adjust its parameters if needed.&lt;br/&gt;Click &lt;span style=&quot; font-weight:600;&quot;&gt;Ok&lt;/span&gt; to import your graph and visualize it using the &lt;span style=&quot; font-weight:600;&quot;&gt;Node Link Diagram&lt;/span&gt; and &lt;span style=&quot; font-weight:600;&quot;&gt;Spreadsheet&lt;/span&gt; views.&lt;br/&gt;&lt;br/&gt;See &lt;span style=&quot; font-weight:600;&quot;&gt;Edit&lt;/span&gt; menu, then &lt;span style=&quot; font-weight:600;&quot;&gt;Preferences &lt;/span&gt;for more options when importing a graph.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="categoryFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>30</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="leftLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="categoryLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;b&gt;Category&lt;/b&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="importModules">
           <property name="styleSheet">
            <string notr="true">QTreeView { color: black; }

 QTreeView::branch {
         border-image: url(none.png) 0;
 }

 QTreeView::branch:has-children:!has-siblings:closed,
 QTreeView::branch:closed:has-children:has-siblings {
         border-image: none;
         image: url(:/tulip/gui/ui/right_arrow-black.png);
 }

 QTreeView::branch:open:has-children:!has-siblings,
 QTreeView::branch:open:has-children:has-siblings  {
         border-image: none;
         image: url(:/tulip/gui/ui/down_arrow-black.png);
 }</string>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="rightFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>80</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="handleWidth">
            <number>1</number>
           </property>
           <property name="childrenCollapsible">
            <bool>false</bool>
           </property>
           <widget class="QFrame" name="parametersFrame">
            <layout class="QVBoxLayout" name="rLayout2">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="parametersLabel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string>&lt;b&gt;Parameters&lt;b&gt;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="parameters">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::AllEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>150</number>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>200</number>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>true</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ImportWizard</sender>
   <signal>currentIdChanged(int)</signal>
   <receiver>ImportWizard</receiver>
   <slot>updateFinishButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>groupSelected(QString)</slot>
  <slot>algorithmSelected(QString)</slot>
  <slot>updateFinishButton()</slot>
 </slots>
</ui>
